gui/wxpython/gui_core/pyedit.py

```diff
@@ -681,6 +688,17 @@ def _set_overwrite(self, overwrite):
     def OnSetParameters(self, *args, **kwargs):
         self.controller.OnSetParameters(*args, **kwargs)
 
+    def OnRecentFilesMenu(self, event):
+        id = event.GetId()
+        menu = event.GetEventObject()
+        label = menu.GetLabelText(id)
+        index = int(label.split(' ')[0]) - 1
```

Comment:
If this is the right way of using wx.FileHistory, this line looks exactly like something we want to encapsulate in the our FileHistory class and not bother with it here.

Reply:
I've already fixed it.